test/juttled/juttled.spec.js

```diff
@@ -85,7 +85,8 @@ describe("Juttled Tests", function() {
             return start_job_check_job_id();
         });
 
-        it("Start 2 jobs at once, should return both jobs", function() {
+        it("Start 2 jobs at once, should return both jobs", function(done) {
+            this.timeout(5000);
```

Comment:
The default timeout I'm seeing is 2s:

Error: timeout of 2000ms exceeded. Ensure the done() callback is being called in this test.